src/Tools/RoslynPublish/RoslynPublish.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\build\Targets\SettingsSdk.props" />
+  <PropertyGroup>
+    <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>RoslynPublish</RootNamespace>
+    <AssemblyName>RoslynPublish</AssemblyName>
+    <TargetFramework>net462</TargetFramework>
+    <RuntimeIdentifiers>win7</RuntimeIdentifiers>
+    <SignAssembly>false</SignAssembly>
+    <AutomaticBindingRedirects>true</AutomaticBindingRedirects>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
```

Comment:
@tmat it is a new SDK project. I asked @davkean, and @srivatsn about whether this was / wasn't necessary anymore. Their repsonse is that it must be included somewhere in the build chain but it can be centralized. Going to do a PR soon to remave all uses of this in Roslyn and centralize it.